plugins/index.js

```diff
@@ -90,8 +91,16 @@ export function loadPlugin(manifest) {
         script.src = getSiteURL() + bundlePath;
         script.onload = onLoad;
         console.log('Loading ' + manifest.id + ' plugin'); //eslint-disable-line no-console
-        document.getElementsByTagName('head')[0].appendChild(script);
 
+        if (!tryRequire(bundlePath)) {
+            console.error('Could not load bundle for ' + manifest.id); //eslint-disable-line no-console
+
+            loadedPlugins[manifest.id] = false;
+            resolve();
```

Comment:
Should we `reject()` here?

Reply:
Yes, for correctness I've updated it to use reject. However, this promise is currently resolved with `.all` where it is called from at [`initializePlugins`](https://github.com/mattermost/mattermost-webapp/blob/cae205cacb7abe574665ba2d4f1a59ec414e1be2/plugins/index.js#L41`).
I wonder if we should keep track of the local state of the plugin? It currently does not know if they were loaded properly. Possible future improvement ?